inference/enterprise/device_manager/metrics_service.py

```diff
@@ -81,18 +91,23 @@ def build_container_stats():
         if container:
             container_stats = {}
             models = aggregate_model_stats(id)
-            container_stats["uuid"] = container.id
+            # TODO: deprecate `uuid` in favor of `alias`
+            container_stats["uuid"] = container.alias
```

Comment:
`uuid` naming got a bit confusing. Since it's the id set by the user, I'm calling it the `alias` in device manager land. It's still mapped to what the value of the `INFERENCE_SERVER_ID` env var (retrieved by the `/info` route)